content/docs/guides/continuous-delivery/spinnaker.md

```diff
@@ -101,98 +126,130 @@ To create this resource in your Spinnaker cluster, run:
 
-* `account` - The account to be used to run the pre-configured job. Add Kubernetes accounts using the [`hal config provider kubernetes add`](https://www.spinnaker.io/reference/halyard/commands/#hal-config-provider-kubernetes-account-add) command.
-* `credentials` - The credentials to use to run the pre-configured job.
-* `namespace` - The namespace in which to run the pre-configured job. Secret resources must also exist in the same namespace.
+1. Create a new Pipeline and click the **Add Stage** button. This will allow you to pick from a list of available stages,
+  including the Pulumi stage which you would have added above.
+1. Configure the inputs for the Pulumi stage accordingly.
+   As previously mentioned in this guide, you can use [pipeline expressions](https://www.spinnaker.io/guides/user/pipeline/expressions/)
+   for your stage inputs.
+1. Set the value of the **Command** input to `preview`.
 
-To set the above configuration properties, open the `~/.hal/config` file where your Halyard installation exists. This typically requires SSH'ing into a Pod if your Spinnaker installation is done using the distributed installation method. Scroll-down to the plugins section and find the `Pulumi.PreConfiguredJobPlugin` section. Under `config:`, configure the above properties as you desire.
+1. Add a [Manual Judgement](https://www.spinnaker.io/guides/tutorials/codelabs/safe-deployments/#adding-a-manual-judgment-to-deployment-pipelines)
+stage to the pipeline and then let's add the Pulumi stage again, but this time we'll set the **Command** input to `up`.
+Set the **Command Args** input to `-s dev --yes`, assuming your stack's name is `dev`. The `--yes` is needed since you
+would be running `pulumi up` non-interactively.
 
-> Don't forget to run `hal deploy apply` after modifying your `~/.hal/config`.
+1. In the end, your pipeline should look like this:
 
-### Per-stage configuration
+    ![Full Pipeline](/images/docs/guides/continuous-delivery/spinnaker/full-pipeline.png)
 
-Navigate to your Spinnaker instance's Deck UI, and in the Pipeline that you wish to run Pulumi, click the `Add Stage` button at the appropriate Pipeline branch and find the `Pulumi` stage.
-The Pulumi stage's UI shows you tooltips to help you fill in the values of the various input properties.
+1. Go back to the main Pipelines view and click on the **Start Manual Execution** link next to your pipeline.
+Once the job starts execution, you should see a **Console Output** link. Clicking it will open a modal that will show
+you the current output.
 
-Spinnaker Pipelines support [pipeline expressions](https://www.spinnaker.io/guides/user/pipeline/expressions/) that allow you to dynamically fill in values of inputs.
+{{% notes "info" %}}
+If you are running Spinnaker on your machine, it can take several minutes before the job starts to execute and the logs are visible. If you see a`ContainerCreating` error, it means the pod running the job isn't ready yet.
+{{% /notes %}}
```

Comment:
We need to indent this note so that the last step doesn't reset back to 1.

```suggestion
    {{% notes "info" %}}
If you are running Spinnaker on your machine, it can take several minutes before the job starts to execute and the logs are visible. If you see a`ContainerCreating` error, it means the pod running the job isn't ready yet.
    {{% /notes %}}
```